src/Workload/Microsoft.Maui.Sdk/Sdk/Microsoft.Maui.Core.Sdk.targets

```diff
@@ -4,6 +4,14 @@
     <UsingMicrosoftMauiCoreSdk>true</UsingMicrosoftMauiCoreSdk>
   </PropertyGroup>
 
+  <!-- Set some property defaults that MAUI requires - especially for dotnet build to work -->
+  <PropertyGroup Condition=" '$([MSBuild]::GetTargetPlatformIdentifier($(TargetFramework)))' == 'windows' ">
+    <EnableMsixTooling Condition=" '$(EnableMsixTooling)' == '' ">true</EnableMsixTooling>
+    <EnablePreviewMsixTooling Condition=" '$(EnablePreviewMsixTooling)' == '' ">$(EnableMsixTooling)</EnablePreviewMsixTooling>
+    <GenerateLibraryLayout Condition=" '$(GenerateLibraryLayout)' == '' and '$(EnableMsixTooling)' == 'true' and '$(OutputType)' != 'Exe' and '$(OutputType)' != 'WinExe' ">true</GenerateLibraryLayout>
+    <WinUISDKReferences Condition=" '$(WinUISDKReferences)' == '' and '$(EnableMsixTooling)' == 'true' ">false</WinUISDKReferences>
+  </PropertyGroup>
```

Comment:
Moved these properties to here to reduce the **required** properties in class libraries that use maui but do not set single project but target windows:
 - we mainly require `EnableMsixTooling` for dotnet build, but it also fixes some bugs
 - `GenerateLibraryLayout` is required for all libraries... not sure why it is not set
 - `WinUISDKReferences` is a compatibility feature and should be avoided in new projects